flocker/common/test/test_interface.py

```diff
@@ -0,0 +1,158 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Tests for ``flocker.common.interface_decorator``.
+"""
+
+from twisted.trial.unittest import SynchronousTestCase
+
+from eliot.testing import (
+    LoggedMessage, assertContainsFields, capture_logging
+)
+from eliot import Field, MessageType
+
+from zope.interface import Interface, implementer
+
+from ..interface import interface_decorator
+
+
+# Eliot structures for testing ``interface_decorator``.
+METHOD = Field.for_types(
+    u"method", [unicode],
+    u"The name of the decorated method.")
+TEST_MESSAGE = MessageType(u"flocker:common:test:interface:message", [METHOD],)
+TEST_EXCEPTION = MessageType(u"flocker:common:test:interface:exception",
+                             [METHOD],)
```

Comment:
I like trailing commas when they simplify diffs as they do when you decide to add another argument to:

```
foo(
    bar,
    baz,
)
```

But these trailing commas won't simplify any diffs, they just look weird. :smile: